docs/strategy-advanced.md

```diff
@@ -20,7 +20,7 @@ It applies a tight timeout for higher priced assets, while allowing more time to
 The function must return either `True` (cancel order) or `False` (keep order alive).
 
 ``` python
-from datetime import datetime, timestamp
+from datetime import datetime
```

Comment:
You're right while being wrong 😆 

timestamp is not used (nor does it exist) - what SHOULD be here is `timedelta` though - as that IS needed and used in the sample below.

Reply:
argh. okay

Reply:
```suggestion
from datetime import datetime, timedelta
```


for simple committing :D